Tests/scripts/collect_tests_and_content_packs.py

```diff
@@ -1071,17 +1069,44 @@ def get_content_pack_name_of_test(tests: set, id_set: Dict = None) -> set:
     return content_packs
 
 
+def get_modified_packs(files_string):
+    modified_packs = set([])
+    all_files = files_string.split('\n')
+
+    for _file in all_files:
+        file_data = _file.split()
+        if not file_data:
+            continue
+        file_status = file_data[0]
+        if file_status.lower().startswith('r'):
+            file_path = file_data[2]
+        else:
+            file_path = file_data[1]
+
+        # ignore files in ".circle", ".github" and ".hooks" directories and .gitignore
+        if not file_path.startswith('.'):
```

Comment:
since you look for specific prefixes (`Base`, `Documentation`), this is redundant.